cfme/tests/networks/nuage/test_events_are_triggered.py

```diff
@@ -120,3 +120,23 @@ def cmp_function(_, full_data):
         event_type=event_type,
         first_event=True
     )
+
+
+def test_targeted_refresh_after_create_entities(register_targeted_refresh, with_nuage_sandbox):
+    """
+    Test whether targeted refresh is triggered when entities are created.
+
+    config:
+        Before running this test you have to configure artifactor(see how in
+        cfme/fixtures/merkyl.py) and make sure you have port 8192 opened
+        on your appliance.
+
+    Steps:
+        * Deploy entities outside of CFME (directly in the provider)
+        * Verify that targeted refreshes where triggered
+    """
+    entities_to_test = ['enterprise']
+    sandbox = with_nuage_sandbox
+
+    for entity in entities_to_test:
+        register_targeted_refresh(entity=entity, ems_ref=sandbox['enterprise'].id)
```

Comment:
`'enterprise'` should not be hardcoded here

Reply:
Nevertheless, this won't work as sandbox keys are not always entity names (look at `cont_vport`, for example). I'd suggest to just do it explicitly for each event we want to capture

```
register_targeted_refresh('enterprise', sandbox['enterprise'].id)
```